switchbot/const.py

```diff
@@ -10,6 +10,14 @@
 DEFAULT_SCAN_TIMEOUT = 5
 
 
+class SwitchbotApiError(RuntimeError):
```

Comment:
The new exception needs to be exported at top level. I'll fix it before I do the release